testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksUnityCompatibility.java

```diff
@@ -145,6 +146,19 @@ public void testManagedTables()
         assertThat(onTrino().executeQuery("SHOW TABLES IN delta." + schemaName)).hasNoRows();
     }
 
+    @Test(groups = {DATABRICKS_UNITY_HTTP_HMS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testManagedSchemas()
```

Comment:
This test case is same as testManagedTables.

Reply:
I'd suggest using `Ignore empty strings for database locations` as a preparatory commit because `testManagedTables()` already handles what we're testing here.